### PR TITLE
Studio V6 Integration

### PR DESCRIPTION
**Pull request by paulr34**
changing session storage depending on what is given from parent iframe

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -249,6 +251,17 @@ export default defineComponent({
     },
   },
   created() {
+    // Parse the query string into the front end.
+    const querystring = require('querystring')
```
Comment (tecimovic) [inline src/App.vue:255]: I don't like stray `require` sitting in the code like this. It should be at the top of the file.

**Review by tecimovic — CHANGES_REQUESTED** [review]
I think there is more to this....

We have logic that allows to attach to the old session. Where is that logic? Is that logic affected?

With that "setItem('session_id')", we're again "spreading around" the session id logic. I would like all logic that touches the session id to be accumulated in one place, not having to hunt it down between the place where we have it no, to here.

File: src/App.vue (hunk 1 above)
Comment (paulr34) [reply src/App.vue:255]: okay will change, thanks

File: src/App.vue [hunk 2]
```diff
@@ -249,6 +251,17 @@ export default defineComponent({
...
+    let search = window.location.search
+
+    if (search[0] === '?') {
+      search = search.substring(1)
+    }
+    let query = querystring.parse(search)
+    if (query[`stsApplicationId`]) {
+      window.sessionStorage.setItem('session_uuid', query[`stsApplicationId`])
```
[the remark this answers is not in the document]
Comment (paulr34) [reply src/App.vue]: okay I addressed this concern, lmk what you think

**Comment by paulr34** [comment]
> I think there is more to this....
> 
> We have logic that allows to attach to the old session. Where is that logic? Is that logic affected?
> 
> With that "setItem('session_id')", we're again "spreading around" the session id logic. I would like all logic that touches the session id to be accumulated in one place, not having to hunt it down between the place where we have it no, to here.

Fixed

**Pull request merged**